shopinvader_partner_firstname/__manifest__.py

```diff
@@ -1,16 +1,13 @@
-# Copyright 2019 Camptocamp SA
+# Copyright 2021 Camptocamp SA
```

Comment:
rollback this please: copyright should always state the 1st here

Reply:
got it :)